src/Data/Nat/Coprimality.agda

```diff
@@ -87,63 +89,63 @@ coprime? i j with mkGCD i j
 -- Nothing except for 1 is coprime to 0.
 
 0-coprimeTo-m⇒m≡1 : ∀ {m} → Coprime 0 m → m ≡ 1
```

Comment:
`m` is now a variable and be removed? Same for `{n}` below?

Reply:
Good spot!